meteor/client/lib/viewPort.ts

```diff
@@ -6,7 +6,7 @@ import { isProtectedString } from '../../lib/lib'
 import RundownViewEventBus, { RundownViewEvents } from '../ui/RundownView/RundownViewEventBus'
 import { Settings } from '../../lib/Settings'
 
-const HEADER_MARGIN = 24 // TODOSYNC: TV2 uses 15. Could this use a css variable and getComputedStyle(document.documentElement).getPropertyValue('--my-variable-name'); ?
+const HEADER_MARGIN = 15 // TODOSYNC: NRK uses 24? do we need to make this better somehow?
```

Comment:
@jstarpl Could you help me with having a look at this, if it's working as intended, or if it needs a change? Feel free to commit any changes you think are necessary.